scripts/xpcs_corr_client.py

```diff
@@ -23,6 +23,8 @@ def arg_parse():
                                 '/A001_Aerogel_1mm_att6_Lq0_001_00001-01000.imm')
     parser.add_argument('--group', help='Visibility in Search', default=None)
     parser.add_argument('--deployment','-d', default='nick-talc', help=f'Deployment configs. Available: {list(deployment_map.keys())}')
+    parser.add_argument('--results', default='/data/xpcs8/2019-1/comm201901/ALCF_results/',
```

Comment:
Maybe the default should go to a dump folder. On the case they submit a "test" but do not pass the flag?

Reply:
Honestly, I'm not sure how to enable this functionality another way. We can currently toggle the transfer state on and off as needed to disable this. The `--results` flag will only produce input on the flow. It would be nice to make the default a dump directory, but I'm not sure where else to put it.